lib/bemxjst/tree.js

```diff
@@ -353,6 +353,10 @@ Tree.prototype.body = function() {
 
 Tree.prototype.match = function() {
   var children = new Array(arguments.length);
+
+  if (!arguments.length)
+    throw new Error('.match() must have argument');
```

Comment:
after gzip:

term 1: `throw new Error('`
term 2: `')`

In your case:

term 1: `assert(something, '`
term 2: `')`

Same shit…